release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -834,6 +851,21 @@ With this release, the image-based upgrade introduces the following enhancements
 
 For more information about the `ImageBasedGroupUpgrade` API, see xref:../edge_computing/image_based_upgrade/ztp-image-based-upgrade.adoc#ztp-image-based-upgrade-concept_ztp-gitops[Managing the image-based upgrade at scale using the ImageBasedGroupUpgrade CR on the hub].
 
+[id="ocp-4-17-disk-encryption_{context}"]
+==== Enabling disk encryption with TPM and PCR protection (Technology Preview)
+
+With this release, you can enable disk encryption with Trusted Platform Module (TPM) and Platform Configuration Registers (PCRs) protection. You can use the `diskEncryption` field in the `SiteConfig` custom resource (CR) to configure the disk encryption. Configuring the `SiteConfig` CR enables disk encryption at the time of cluster installation.
+
+For more information, see xref:../edge_computing/ztp-reference-cluster-configuration-for-vdu.adoc#ztp-sno-du-configuring-disk-encryption-with-pcr-protection_sno-configure-for-vdu[Enabling disk encryption with TPM and PCR protection]
+
+[id="ocp-4-17-config-ipsec-encryption_{context}"]
+==== Configuring IPsec encryption for multi-node clusters using {ztp} and SiteConfig resources
```

Comment:
Same here - consider re-titling to "About IPsec encryption...".

Reply:
Updated as per the suggestion.